docs/orleans/grains/observers.md

```diff
@@ -1,38 +1,26 @@
 ---
 title: Observers
 description: Learn about observers in .NET Orleans.
-ms.date: 01/31/2022
+ms.date: 03/16/2022
 ---
 
 # Observers
 
 There are situations in which a simple message/response pattern is not enough, and the client needs to receive asynchronous notifications.
 For example, a user might want to be notified when a new instant message has been published by a friend.
 
-Client observers is a mechanism that allows notifying clients asynchronously.
-An observer is a one-way asynchronous interface that inherits from `IGrainObserver`, and all its methods must be void.
-The grain sends a notification to the observer by invoking it like a grain interface method, except that it has no return value, and so the grain need not depend on the result.
-The Orleans runtime will ensure one-way delivery of the notifications.
-A grain that publishes such notifications should provide an API to add or remove observers.
-In addition, it is usually convenient to expose a method that allows an existing subscription to be cancelled.
+Client observers is a mechanism that allows notifying clients asynchronously. An observer is a one-way asynchronous interface that inherits from <xref:Orleans.IGrainObserver>, and all its methods must be void. The grain sends a notification to the observer by invoking it like a grain interface method, except that it has no return value, and so the grain need not depend on the result. The Orleans runtime will ensure one-way delivery of the notifications. A grain that publishes such notifications should provide an API to add or remove observers. In addition, it is usually convenient to expose a method that allows an existing subscription to be cancelled.
 
-Grain developers may use a utility class such as [`ObserverManager<T>`](https://github.com/dotnet/orleans/blob/e997335d2d689bb39e67f6bcf6fd70862a22c02f/test/Grains/TestGrains/ObserverManager.cs#L12) to simplify development of observed grain types.
-Unlike grains, which are automatically reactivated as-needed after failure, clients are not fault tolerant: a client which fails may never recover.
-For this reason, the `ObserverManager<T>` utility removes subscriptions after a configured duration.
-Clients which are active should resubscribe on a timer to keep their subscription active.
+Grain developers may use a utility class such as [`ObserverManager<T>`](https://github.com/dotnet/orleans/blob/e997335d2d689bb39e67f6bcf6fd70862a22c02f/test/Grains/TestGrains/ObserverManager.cs#L12) to simplify development of observed grain types. Unlike grains, which are automatically reactivated as-needed after failure, clients are not fault-tolerant: a client which fails may never recover.
```

Comment:
I think these were the permalink, and intentional so that if the line number changes in the future - this conceptual discussion about a specific member is still valid.